dbfit-java/informix/src/main/java/dbfit/environment/InformixBigDecimalTransformer.java

```diff
@@ -0,0 +1,21 @@
+package dbfit.environment;
+
+import java.sql.SQLException;
+
+import dbfit.util.TypeTransformer;
+
+public class InformixBigDecimalTransformer implements TypeTransformer {
+
+    @Override
+    public Object transform(Object o) throws SQLException {
+        if (o == null) {
+            return null;
+        }
+        if (!(o instanceof java.math.BigDecimal)) {
+            throw new SQLException(this.getClass().getSimpleName() +
```

Comment:
I've reverted it to `UnsupportedOperationException`. It's still not clear which classes could actually throw a `SQLException` though (as per the specification). May be we can revisit this another day.